prep-javascript-objects/main.js

```diff
@@ -0,0 +1,16 @@
+let person = {
+  firstName: 'Summley',
+  lastName: 'Phung',
+  hobby: 'basketball',
+};
+
+let fullName = person.firstName + ' ' + person.lastName;
+console.log("The person's name is: ", fullName);
+
+person.job = 'Motion Designer';
+console.log("The person's current job is: ", person.job);
+
+person.previousJob = 'Photographer';
```

Comment:
Bracket notation should be used here
`person['previousJob'] = 'Photographer';`

Reply:
What's the reason that bracket notation should be used here instead of dot notation?

Reply:
The only reason to use bracket notation in this case is because the instructions required it. Either approach would work.